src/containers/BooksList.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Book from '../components/Book'
+
+
+
+
+class BooksList extends React.Component {
```

Comment:
`BooksList` doesn't need local `state` nor any component lifecycle methods. Therefore, we should just keep it as a functional component and not `class`